### ISSUE TITLE
instagram: publishing likes sometimes 400s

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy#341]
seeing this for @barryf (sorry barry!). examples on https://www.brid.gy/instagram/barryf#publishes

@kylewm has been debugging. he's seeing instagram complain _ OAuthPermissionsException: This request requires scope=likes, but this access token is not authorized with this scope. The user must re-authorize your application with scope=likes to be granted write permissions._

...weird though, since barryf definitely enabled publish and re-authed to instagram with `scope=likes`, and the same dance worked for us. hey @barryf, did anything unusual happen when you enabled publish for instagram on bridgy?

**Comment by barryf** [comment]
Hey Ryan. I've just disabled and re-enabled publishing in Bridgy and likes are working again perfectly, e.g. https://barryfrost.com/2015/01/11-084931

I tried IG likes when I saw your posts in IRC a few days ago and all was fine, but yesterday I was hacking together like-contexts for my site and publishing to Bridgy started 400'ing. I wasn't sure if it was something I'd caused but re-authorizing today seems to have solved it. I'll let you know if it wobbles again.

Thanks guys for your work on this feature -- it's great having likes published on my site.
